Mollie.Api/Models/Chargeback/ChargebackResponse.cs

```diff
@@ -32,7 +32,12 @@ public class ChargebackResponse : IResponseObject {
         /// <summary>
         /// The id of the payment this chargeback belongs to.
         /// </summary>
-		public string PaymentId { get; set; }
+	public string PaymentId { get; set; }
+	    
+	/// <summary>
+        /// The reason given for a Chargeback, this can help determine the cost for the chargeback
+        /// </summary>
+        public ChargebackResponse ChargebackResponseReason { get; set; }
```

Comment:
This property should be of type `ChargebackResponseReason` not `ChargebackResponse` right?